adapters/handlers/rest/configure_api.go

```diff
@@ -1422,6 +1424,14 @@ func registerModules(appState *state.State) error {
 			Debug("enabled module")
 	}
 
+	if _, ok := enabledModules[modmulti2multivecweaviate.Name]; ok {
```

Comment:
please add it to `defaultVectorizers` list as we did it with `text2vec-weaviate` module